From 3fcb74ea6c498185bcbc6e6adbdc8189f1ed1767 Mon Sep 17 00:00:00 2001
From: Philipp Trulson <philipp@trulson.de>
Date: Mon, 22 Oct 2018 16:17:43 +0200
Subject: [PATCH] Disable compiling errors

---
 buildSrc/linux.gradle | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/buildSrc/linux.gradle b/buildSrc/linux.gradle
index b87e22b..1600771 100644
--- a/buildSrc/linux.gradle
+++ b/buildSrc/linux.gradle
@@ -141,7 +141,7 @@ LINUX.glass.javahInclude = [
     "com/sun/glass/ui/gtk/*"]
 LINUX.glass.nativeSource = file("modules/graphics/src/main/native-glass/gtk")
 LINUX.glass.compiler = compiler
-LINUX.glass.ccFlags = [ccFlags, "-Werror"].flatten()
+LINUX.glass.ccFlags = [ccFlags, "-Wno-error"].flatten()
 LINUX.glass.linker = linker
 LINUX.glass.linkFlags = [linkFlags].flatten()
 LINUX.glass.lib = "glass"
-- 
2.19.1


